Notebook.ipynb

```diff
@@ -4,7 +4,28 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Penney's Game "
+    "# Penney's Game "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----------------"
+   ]
+  },
+  {
```

Comment:
I don't like this block. Either remove it or merge it with the previous one. It feels like there were something there which we removed and forgot to clean the rest.

Reply:
fixed in https://github.com/sepandhaghighi/penney/pull/60/commits/6964fcaefdcc4d2fccb0145baa4588073dbb3bb3